steps/steps.go

```diff
@@ -39,8 +40,20 @@ func stepListItem(stepIDComposite, title, runIf string, inputs ...envmanModels.E
 func DefaultPrepareStepList(params PrepareListParams) []bitriseModels.StepListItemModel {
 	stepList := []bitriseModels.StepListItemModel{}
 
-	if params.ShouldIncludeActivateSSH {
-		stepList = append(stepList, ActivateSSHKeyStepListItem(""))
+	switch params.RepoAccess {
+	case models.RepoAccessPublic: // No SSH key setup needed
```

Comment:
Would it make sense to remove this line and just leave the same comment (`// No SSH key setup needed for public repos`) on top of the switch statement? It tricked me because first I though that the empty case will fall through to the second and I had to google how Go behaves to realise that will not happen. But it made me do this extra work which I think others might go through and could be avoided. 😄

Reply:
We have the `exhaustive` linter enabled in this project (for good reason, I think it's very useful!), so I don't want to completely remove this case, but I can wrap the comment in curly braces, take a look now.